src/core/searchUtils.js

```diff
@@ -92,8 +92,28 @@ export function loadSearchResultsIfNeeded({ store: { dispatch, getState }, locat
     ...location.query,
     clientApp: state.api.clientApp,
   });
+
   if (!isLoaded({ state: state.search, page, filters })) {
     return performSearch({ dispatch, page, api: state.api, auth: state.auth, filters });
   }
   return true;
 }
+
+export function loadByCategoryIfNeeded(
```

Comment:
Is this making another API request on the client after JS loading on initial render? I think that's why we've normally included the caching.